task-maker-iospec/tests/goldenfiles/segtree/IOSPEC

```diff
@@ -0,0 +1,56 @@
+inputln {
+    item n: i32;
+    item q: i32;
+}
+
+inputln {
+    for i upto n {
+        item a[i]: i32;
+    }
+}
+
+// @call init(a=a);
```

Comment:
Why are all the `@call` in this spec commented out? Is this the reason why the solution files are empty?